package.json

```diff
@@ -3,6 +3,14 @@
   "version": "1.7.1",
   "description": "Security rules for eslint",
   "main": "index.js",
+  "files": [
+    "docs/",
+    "rules/",
+    "utils/",
+    "configs/",
+    "index.js",
+    "CHANGELOG.md"
+  ],
```

Comment:
adding the field also decrease the installation size. :)

Reply:
The following paths and files are never ignored, so adding them to .npmignore is pointless:

* package.json
* README (and its variants)
* CHANGELOG (and its variants)
* LICENSE / LICENCE

so I will remove `changelog.md` for consistency. :)

Reply:
@aladdin-add Per https://docs.npmjs.com/cli/v9/configuring-npm/package-json#files I'm not sure if CHANGELOG and variants is included by default (although it might be a quick local test)
> Certain files are always included, regardless of settings:
> - `package.json`
> - `README`
> - `LICENSE` / `LICENCE`
> - The file in the "main" field
>
> `README` & `LICENSE` can have any case and extension.

Reply:
removing both `changelog` and `license`, I've tested in my pc, here is the output:
```bash
➜  eslint-plugin-security git:(feat/new-config) npm pack
npm notice 
npm notice 📦  eslint-plugin-security@1.7.1
npm notice === Tarball Contents === 
npm notice 11.3kB LICENSE                                            
npm notice 5.3kB  README.md                                          
npm notice 263B   configs/recommended.js                             
npm notice 4.1kB  docs/avoid-command-injection-node.md               
npm notice 1.8kB  docs/bypass-connect-csrf-protection-by-abusing.md  
npm notice 4.5kB  docs/regular-expression-dos-and-node.md            
npm notice 2.0kB  docs/rules/detect-bidi-characters.md               
npm notice 433B   docs/rules/detect-buffer-noassert.md               
npm notice 505B   docs/rules/detect-child-process.md                 
npm notice 422B   docs/rules/detect-disable-mustache-escape.md       
npm notice 417B   docs/rules/detect-eval-with-expression.md          
npm notice 215B   docs/rules/detect-new-buffer.md                    
npm notice 468B   docs/rules/detect-no-csrf-before-method-override.md
npm notice 353B   docs/rules/detect-non-literal-fs-filename.md       
npm notice 352B   docs/rules/detect-non-literal-regexp.md            
npm notice 424B   docs/rules/detect-non-literal-require.md           
npm notice 313B   docs/rules/detect-object-injection.md              
npm notice 237B   docs/rules/detect-possible-timing-attacks.md       
npm notice 244B   docs/rules/detect-pseudoRandomBytes.md             
npm notice 342B   docs/rules/detect-unsafe-regex.md                  
npm notice 6.6kB  docs/the-dangers-of-square-bracket-notation.md     
npm notice 2.6kB  index.js                                           
npm notice 1.9kB  package.json                                       
npm notice 3.1kB  rules/detect-bidi-characters.js                    
npm notice 2.0kB  rules/detect-buffer-noassert.js                    
npm notice 2.2kB  rules/detect-child-process.js                      
npm notice 870B   rules/detect-disable-mustache-escape.js            
npm notice 964B   rules/detect-eval-with-expression.js               
npm notice 684B   rules/detect-new-buffer.js                         
npm notice 1.4kB  rules/detect-no-csrf-before-method-override.js     
npm notice 3.3kB  rules/detect-non-literal-fs-filename.js            
npm notice 1.3kB  rules/detect-non-literal-regexp.js                 
npm notice 1.3kB  rules/detect-non-literal-require.js                
npm notice 2.3kB  rules/detect-object-injection.js                   
npm notice 1.9kB  rules/detect-possible-timing-attacks.js            
npm notice 1.0kB  rules/detect-pseudoRandomBytes.js                  
npm notice 1.7kB  rules/detect-unsafe-regex.js                       
npm notice 982B   utils/data/fsFunctionData.json                     
npm notice 477B   utils/find-variable.js                             
npm notice 6.2kB  utils/import-utils.js                              
npm notice 7.1kB  utils/is-static-expression.js                      
npm notice === Tarball Details === 
npm notice name:          eslint-plugin-security                  
npm notice version:       1.7.1                                   
npm notice filename:      eslint-plugin-security-1.7.1.tgz        
npm notice package size:  22.7 kB                                 
npm notice unpacked size: 83.7 kB                                 
npm notice shasum:        0ff6502beca7109302885b00cdc82936d4323e56
npm notice integrity:     sha512-QVETGaC4a4yrh[...]pM8xozv/LGxbA==
npm notice total files:   41                                      
npm notice 
eslint-plugin-security-1.7.1.tgz
```

looks like we still need to add `changelog`. :)